openedx/core/djangoapps/course_groups/tests/test_views.py

```diff
@@ -125,7 +125,7 @@ def create_cohorted_discussions(self):
             self.course,
             is_cohorted=True,
             discussion_topics=discussion_topics,
-            cohorted_discussions=cohorted_discussions
+            divided_discussions=cohorted_discussions
```

Comment:
Nit: Can cohorted_discussions be renamed also?

This might not matter as I think I am updating the names in this method in my PR off of this branch.